### PR TITLE
Modified notebook forecasting-retail-demand

### PR DESCRIPTION
**Pull request by sudarshan-SpringML**
Modified notebook forecasting-retail-demand
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

**Comment by sudarshan-SpringML** [comment]
ran fine in workbench and colab

File: notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb [hunk 1]
```diff
@@ -0,0 +1,5688 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p>Combine title and links into single cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p>remove Table of Contents</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p>move Dataset below Objective</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p>use template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p><strong>Google Cloud Notebooks -&gt; Vertex AI Workbench Notebooks</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p>Combine with above text cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p>Use template version</p><p> </p><p>Pip installs all in single cell</p><p> </p><p>Remove execution output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p>Use template version for PROJECT_ID</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p><strong>Google Cloud Notebooks -&gt; Vertex AI Workbench Notebooks</strong></p><p> </p><p>Combine these two text cells into one</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p>use latest template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p>Code cell needs accompanying text cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p>Is this executable?</p><p> </p><p>if not, remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p>Is this executable?</p><p>if not, remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p>Is this executable?</p><p>if not, remove</p><p> </p><p>remove execution output from throughout the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (andrewferlitsch) [inline notebooks/official/workbench/demand_forecasting/forecasting-retail-demand.ipynb:1]: <p>Is this executable?</p><p>if not, remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/827/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

**Comment by sudarshan-SpringML** [comment]
@andrewferlitsch all changes done

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**